final-project/scripts/blog.js

```diff
@@ -0,0 +1,47 @@
+import api from './api';
+import Slider from './slider';
+import { $ } from './utils';
+
+class Blog {
+  constructor() {
+    this.sliderContainer = $('#blog-slider');
+  }
+
+  createSlides(posts) {
+    return posts.map(({description, images, title, url}, index) => (
+      `
```

Comment:
Another way of doing this is by cloning an existing HTML node and adding the data to it, that way you in case the HTML changes you won't need to update your JS, you just need to have good selectors.